constants/values.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path')
```

Comment:
We have to be careful here and make sure we don't import any backend libraries into frontend repos.
* It's fine that this is here, and I think it's a great idea to do this; but make sure it's only used by repos that have access to node.

The taps frontend code should never try to import this.
One thing that might help is use `module-alias` to define shortcuts, but also as a way of defining backend vs frontend files / locations

Your shortcut could include something like `$BackConstants` vs `$FrontConstants`, or something like

Reply:
Updated: split the constants so that there are two module-aliases: `HerkinFrontConstants` and `HerkinBackConstants`